frontend/Peerprep/components/navigation/Navbar.vue

```diff
@@ -29,7 +29,7 @@ const getInitials = () => {
                 <div class="flex items-center space-x-6">
                     <NavLink to="/" exact>Home</NavLink>
                     <NavLink to="/questions">Questions</NavLink>
-                    <!-- <NavLink to="/collaboration">Collaboration</NavLink> -->
+                    <NavLink to="/collaboration">Collaboration</NavLink>
```

Comment:
I think we should still not allow users to navigate to collaboration via navbar if they are being redirected via a match

Reply:
removed link to collaboration from navbar and added redirect from /collaboration to / when collabstore is empty is middleware